personalized_nlp/models/baseline_tune.py

```diff
@@ -0,0 +1,21 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+class NetTune(nn.Module):
```

Comment:
Here we have a issue which can somehow explain the fine-tuned results for the baseline model. 

In the case of transformer model finetuining is about learning (finetuning) transformer model to get better text embeddings from it -- here we do nat have it. Here, we have only added `frozen` parameter which is used in `Classifier` or `Regressor` class but in fact there is no tranformer model which is frozen/unfrozen so we still use the oroginal embeddings forom pretrained model.

`Net` and `NetTune` class should be merged together, there should be initilized transformer model which depending on value of `frozen` should be used or not in `foraward()` method. In short, it should be similar as it is in `NetUserID` class in `personalized_nlp/models/user_id.py`.

So unfortunatelly, these fine-tuned baseline model wasnot working as it was expected, and was working differently than the other fine-tuned methods in the paper.